bin/metrics-iostat-extended.rb

```diff
@@ -85,7 +85,7 @@ def parse_results(raw)
         headers = line.gsub(/%/, 'pct_').split(/\s+/)
         headers.shift
         next
-      when /^(Device):/
+      when /^(Device).*/
```

Comment:
To be a bit more specific we could use `when /^(Device):\s+.+/, /^(Device)\s+.+/`